doc/rst/language/spec/procedures.rst

```diff
@@ -1165,7 +1165,7 @@ listed in the following table:
 arity    operators
 ======== ===============================
 unary    ``&`` ``+`` ``-`` ``!`` ``~``
```

Comment:
What's this unary `&` operator?  The compiler lets me define one on a record, but I can't call it as either prefix or postfix.

Reply:
I opened #17337 for these questions as they're off-topic here.